queue_services/entity-filer/src/entity_filer/filing_processors/cease_receiver.py

```diff
@@ -0,0 +1,31 @@
+# Copyright © 2025 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""File processing rules and actions for the cease receiver."""
+from typing import Dict
+
+from legal_api.models import Business, Filing
+
+from entity_filer.filing_meta import FilingMeta
+from entity_filer.filing_processors.filing_components.parties import update_parties
+
+
+def process(business: Business, filing: Dict, filing_rec: Filing, filing_meta: FilingMeta):
+    # pylint: disable=too-many-branches;
+    """Render the cease_receiver onto the business model objects."""
+    cease_receiver_filing = filing.get('ceaseReceiver')
+    if not cease_receiver_filing.get('parties'):
+        return
+
+    if parties := cease_receiver_filing.get('parties'):
+        update_parties(business, parties, filing_rec, False)
```

Comment:
This won't work. You can refer correction filing https://github.com/bcgov/lear/blob/main/queue_services/entity-filer/src/entity_filer/filing_processors/filing_components/correction.py#L159